PoC/SpringBootProjectTemplate/src/main/java/com/devglan/SpringBootProjectTemplateApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.devglan;
```

Comment:
These package names too can be changed to match our repo
- ie.tcd.springtemplate.*

Reply:
Done